webapp/packages/core-blocks/src/FormControls/Input.tsx

```diff
@@ -10,23 +10,23 @@ import React, { forwardRef, useCallback, useId, useLayoutEffect, useRef, useStat
 
 import { isNotNullDefined } from '@dbeaver/js-helpers';
 
-import { filterLayoutFakeProps, getLayoutProps } from '../../Containers/filterLayoutFakeProps.js';
-import type { ILayoutSizeProps } from '../../Containers/ILayoutSizeProps.js';
-import { Icon } from '../../Icon.js';
-import { useTranslate } from '../../localization/useTranslate.js';
-import { s } from '../../s.js';
-import { useCombinedHandler } from '../../useCombinedHandler.js';
-import { useCombinedRef } from '../../useCombinedRef.js';
-import { useS } from '../../useS.js';
-import { useStateDelay } from '../../useStateDelay.js';
-import { Field } from '../Field.js';
-import { FieldDescription } from '../FieldDescription.js';
-import { FieldLabel } from '../FieldLabel.js';
-import { useCapsLockTracker } from '../useCapsLockTracker.js';
-import inputFieldStyle from './InputField.module.css';
-import { IconButton, Input, Spinner, type InputProps } from '@dbeaver/ui-kit';
+import { filterLayoutFakeProps, getLayoutProps } from '../Containers/filterLayoutFakeProps.js';
+import type { ILayoutSizeProps } from '../Containers/ILayoutSizeProps.js';
+import { Icon } from '../Icon.js';
+import { useTranslate } from '../localization/useTranslate.js';
+import { s } from '../s.js';
+import { useCombinedHandler } from '../useCombinedHandler.js';
+import { useCombinedRef } from '../useCombinedRef.js';
+import { useS } from '../useS.js';
+import { useStateDelay } from '../useStateDelay.js';
+import { Field } from './Field.js';
+import { FieldDescription } from './FieldDescription.js';
+import { FieldLabel } from './FieldLabel.js';
+import { useCapsLockTracker } from './useCapsLockTracker.js';
+import inputStyle from './Input.module.css';
+import { IconButton, Input as AriakitInput, Spinner, type InputProps as UiKitInputProps } from '@dbeaver/ui-kit';
```

Comment:
I guess it's better to use the same naming for Component and ComponentProps for consistency.